packages/components/src/local/molecules/forces-in-channel/index.tsx

```diff
@@ -21,8 +21,9 @@ const useButtonStyle = makeStyles(() =>
   })
 )
 
-export const ForcesInChannel: React.FC<Props> = ({ icons, onMarkAllAsRead }: Props) => {
+export const ForcesInChannel: React.FC<Props> = ({ icons, colors, onMarkAllAsRead }: Props) => {
   const buttonStyles = useButtonStyle()
+  console.log(icons, colors)
```

Comment:
@zukilover  - could I have a helping hand please?

The icons need to have a background shade to match their force.  I've collated the colors in the `colors` array, and it's got through to here. But, I don't know how to actually shade the icon.

Could you finish it off for me please?

Reply:
I have implemented force's colors as the icon border color. Pls check on the latest commit.

Reply:
https://github.com/serge-web/serge/pull/499/commits/0762af2495f94778f694328070ededdeb3dd02bc

Reply:
Thanks @zukilover - but the icons themselves aren't visible.

I guess the underlying problem is that the icon is black against a dark-blue background.  So, they would be more visible if the background was in the force color (not just the border)

![image](https://user-images.githubusercontent.com/1108513/95309470-16baa200-0883-11eb-8f32-85378078c13e.png)

Reply:
Was able to make change:
![image](https://user-images.githubusercontent.com/1108513/95311161-276c1780-0885-11eb-9302-d5044a60e938.png)

Reply:
Awesome. I was thinking about having svg icons for them, but I think background colors did the job.